### PR TITLE
creating components folder and tester file

### PR DESCRIPTION
**Pull request by jpatrick7926**
Hey guys, just finished creating the components folder and added a test file. This is mainly to show Juan how different branches and PRs work. Please give feedback and let me know if it is good to merge. :)

### CODE REVIEW

File: src/components/test.jsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const testFunct = () => {
```
Comment (Juanmarin444) [inline src/components/test.jsx:1]: you should change testFunct to testFunc before you merge.

**Review by Juanmarin444 — APPROVED** [review]
Yeah looks gooooooooooooood merge mofo! :D

**Comment by jpatrick7926** [comment]
Thanks for the feedback everyone. The thang is merged!

**Comment by Juanmarin444** [comment]
NP MAIIIIIIIN

**Pull request merged**